applications/asset_tracker/src/cloud_codec/cloud_codec.c

```diff
@@ -0,0 +1,385 @@
+/*
+ * Copyright (c) 2019 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+ */
+
+
+#include <stdbool.h>
+#include <string.h>
+#include <zephyr.h>
+#include <zephyr/types.h>
+
+#include "cJSON.h"
+#include "cJSON_os.h"
+#include "cloud_codec.h"
+
+#include <logging/log.h>
+
+LOG_MODULE_REGISTER(cloud_codec, CONFIG_NRF_CLOUD_LOG_LEVEL);
```

Comment:
I think the log level should have its own Kconfig option and not reuse nRF Cloud level

Reply:
I am leaning towards removing the log level config entirely as that seems to be common within NCS.